src/components/edit-asset-address-form/editable-address/editable-address.tsx

```diff
@@ -134,7 +102,14 @@ export const EditableAddress = ({
       });
   };
 
-  if (!llpgAddress && loading) {
+  const getFormInitialValues = () => {
+    if (llpgAddress && assetHasUprn) {
+      return getLlpgAddressFormValues(llpgAddress);
+    }
+    return getCurrentAddressFormValues(currentAddress);
+  };
```

Comment:
Refer to the previous comment. These methods do the same thing

Reply:
They work similarly but don't do the same thing, they take parameters of different types